packages/jsii-spec/lib/spec.ts

```diff
@@ -180,10 +180,80 @@ export interface PackageVersion {
 
 /**
  * Key value pairs of documentation nodes.
- * Based on JSDoc.
+ * Based on TSDoc.
  */
 export interface Docs {
-    [key: string]: string;
+    /**
+     * Summary documentation for an API item.
+     *
+     * The first part of the documentation before hitting a `@remarks` tags, or the first
+     * line of the doc comment block if there is no `@remarks` tag.
+     */
+    summary?: string;
+
+    /**
+     * Detailed information about an API item.
+     *
+     * Either the explicitly tagged `@remarks` section, otherwise everything past the
+     * first paragraph if there is no `@remarks` tag.
+     */
+    remarks?: string;
+
+    /**
+     * If present, this block indicates that an API item is no longer supported and may be
+     * removed in a future release.  The `@deprecated` tag must be followed by a sentence
+     * describing the recommended alternative.  Deprecation recursively applies to members
+     * of a container.  For example, if a class is deprecated, then so are all of its members.
```

Comment:
Are we propagating this in the jsii manifest? (we should)

Reply:
I've been going back and forth on this, and ultimately decided that the assembly would map most closely to the source of truth (the source code). The propagation analysis is done by `jsii-reflect`.

Reply:
Sounds good